app/assets/stylesheets/searchworks4/bookmarks.css

```diff
@@ -118,3 +118,17 @@
     --bl-icon-color: var(--stanford-cardinal);
   }
 }
+
+.toggle-bookmark-remove-all {
+  --bs-btn-active-color: rgb(var(--stanford-digital-red-light-rgb));
+  --bs-btn-border-color: rgb(var(--stanford-digital-red-light-rgb));
+  --bs-btn-color: rgb(var(--stanford-digital-red-light-rgb));
+  --bs-btn-hover-color: rgb(var(--stanford-digital-red-light-rgb));
+  --bs-btn-hover-border-color: rgb(var(--stanford-digital-red-light-rgb));
+  --bs-btn-active-color: rgb(var(--stanford-digital-red-light-rgb));
+  --bs-btn-active-border-color: rgb(var(--stanford-digital-red-light-rgb));
```

Comment:
I'm not seeing these defined in the figma, but I wonder if the normal + active/hover states ought to be visually different?